ui-admin/src/study/participants/KitRequests.tsx

```diff
@@ -56,6 +40,26 @@ export default function KitRequests({ enrollee, studyEnvContext, onUpdate }:
     onUpdate()
   }
 
+  const columns: ColumnDef<KitRequest, string>[] = [{
```

Comment:
style nit -- I think this can and should be declared outside of the component -- it doesn't change based on the data